CLI/my_cli.py

```diff
@@ -0,0 +1,28 @@
+import asyncio
+import time
+from typing import Dict, List
+from Parser.pptx_parser import PptxParser
+from Explainer.gpt_explainer import get_explanations_for_dict
+
+def print_nicely(explanations: Dict[int, List[str]]) -> None:
+    formatted_explanations = "\n".join(
+        f"Page [{page}]\nContent: {content}\nExplanation:\n{explanation}\n"
+        for page, (content, explanation) in ((page, (content, explanation.replace('.', '.\n'))) for page, (content, explanation) in explanations.items())
+    )
+    print(formatted_explanations)
+
+
+async def main() -> None:
+    start_time = time.time()
+
+    pptx_file_path = "../Test/files/presentation_for_tst.pptx"
```

Comment:
You should get the path from the user

Reply:
It just a main for  personal tests .. but ok Ill change it